test/all/index.js

```diff
@@ -208,6 +208,15 @@ test('makePublic', function(t) {
   })
 });
 
+test('updateBucket', function(t) {
```

Comment:
Ahh I was afraid of that. We have these options to test updating.

pubkeys - Array
encryptionKey - String
publicPermissions - Array

So i'll just changing this to update a pubkey.